cache/cache.go

```diff
@@ -36,7 +36,7 @@ func (e ErrKeyNotFound) Error() string {
 type Cache struct {
 	options     []CacheOption
 	cache       map[string]any
-	redisCon    *redis.Client
+	redisCon    *redis.Cmdable
```

Comment:
`Cmdable` is interface so no pointer is needed, just as later on when using it does not need to be used to get address